applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.hpp

```diff
@@ -1158,16 +1135,10 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellCrossSection : public Fl
     	mStorePlyConstitutiveMatrices = true;
     	mPlyConstitutiveMatrices = std::vector<Matrix>(this->NumberOfPlies());
 
-    	for (unsigned int ply = 0; ply < this->NumberOfPlies(); ++ply)
+    	for (IndexType ply = 0; ply < this->NumberOfPlies(); ++ply)
     	{
-    		if (mBehavior == Thick)
-    		{
-    			mPlyConstitutiveMatrices[ply].resize(8, 8, false);
-    		}
-    		else
-    		{
-    			mPlyConstitutiveMatrices[ply].resize(6, 6, false);
-    		}
+    		if (mBehavior == Thick) mPlyConstitutiveMatrices[ply].resize(8, 8, false);

```

Comment:
Minor formatting issue. I don't think this inline is the normal way to format if statements.

Reply:
I like it more this way :)
Can change it though if it is not the standard

Reply:
ok, it's minor so just leave it.